server/functional-test/src/test/java/org/zanata/feature/document/UploadTest.java

```diff
@@ -50,15 +49,14 @@
     public CleanDocumentStorageRule documentStorageRule =
             new CleanDocumentStorageRule();
     private TestFileGenerator testFileGenerator = new TestFileGenerator();
-    private String documentStorageDirectory;
 
     @Before
     public void before() {
         new BasicWorkFlow().goToHome().deleteCookiesAndRefresh();
         new LoginWorkFlow().signIn("admin", "admin");
         new ZanataRestCaller().createProjectAndVersion("uploadtest",
                 "txt-upload", "file");
-        documentStorageDirectory = CleanDocumentStorageRule
+        String documentStorageDirectory = CleanDocumentStorageRule
                 .getDocumentStoragePath().concat(File.separator)
                 .concat("documents").concat(File.separator);
         if (new File(documentStorageDirectory).exists()) {
```

Comment:
Again, it might be better just to clean up here, rather than risk a failed test.